go.mod

```diff
@@ -3,6 +3,8 @@ module github.com/pomerium/pomerium
 go 1.12
 
 require (
+	github.com/cnf/structhash v0.0.0-20180104161610-62a607eb0224
```

Comment:
I'm a little bit concerned about cnf/structhash just because it looks unmaintained. 

Maybe you already considered it, but I'm curious what you think of using  https://github.com/mitchellh/hashstructure which (I believe) is identical in functionality and is a dependency/funded by hashicorp products.

Reply:
Ah.  Yes I'm open to switching back if hashicorp uses hashstructure.  I had some issues with getting consistent hashes from it originally and tried swapping libraries just to isolate the cause.  

I got it working after the switch and didn't consider going back.  Easy change IIRC.